### ISSUE TITLE
Some Readme is missing

### ISSUE THREAD

**Issue by Huge** [coala-analyzer/coala-gui#12]
It would be nice for this repository/app to have a Readme.md or something with a concise instruction on how to install and use the GUI app.

**Comment by sils** [comment]
Yeah so this is a bit of an abandoned project, we should add a note :/ @Udayan12167 can you add a note?

**Comment by AbdealiLoKo** [comment]
Is this abandoned in favour of something else ? Or put on hold in coala's roadmap ?

**Comment by sils** [comment]
This is abandoned in favour of coala-html which is simpler and works
platform independent, we should add a note. I already added it in project
description yesterday, readme should get a huge warning as well

On Oct 31, 2016 3:48 AM, "AbdealiJK" notifications@github.com wrote:

Is this abandoned in favour of something else ? Or put on hold in coala's
roadmap ?

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
https://github.com/coala/coala-gui/issues/12#issuecomment-257203267, or mute
the thread

**Comment by Adrianzatreanu** [comment]
talking about coala html..
